tests/Microsoft.Bot.Builder.Dialogs.Declarative.Tests/SchemaMergeTests.cs

```diff
@@ -57,17 +55,35 @@ static SchemaMergeTests()
                 ProcessStartInfo startInfo;
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 {
+                    var args = $"/C bf dialog:merge ../../libraries/**/*.schema ../../libraries/**/*.uischema ../**/*.schema !../**/testbot.schema -o {schemaPath}";
+                    var oldSchema = File.Exists(schemaPath) ? File.ReadAllText(schemaPath) : string.Empty;
                     File.Delete(schemaPath);
-                    startInfo = new ProcessStartInfo("cmd.exe", $"/C bf dialog:merge ../../libraries/**/*.schema ../../libraries/**/*.uischema ../**/*.schema !../**/testbot.schema -o {schemaPath}");
+                    startInfo = new ProcessStartInfo("cmd.exe", args);
                     startInfo.WorkingDirectory = projectPath;
                     startInfo.UseShellExecute = false;
                     startInfo.CreateNoWindow = true;
+                    startInfo.RedirectStandardOutput = true;
                     startInfo.RedirectStandardError = true;
 
                     var process = Process.Start(startInfo);
                     var error = process.StandardError.ReadToEnd();
                     process.WaitForExit();
 
+                    var newSchema = File.Exists(schemaPath) ? File.ReadAllText(schemaPath) : string.Empty;
+                    if (error.Length != 0 || !newSchema.Equals(oldSchema))
+                    {
+                        // Try installing latest bf if the schema changed
+                        startInfo.Arguments = $"/C npm i -g @microsoft/botframework-cli@next";
```

Comment:
This assumes npm is installed.

Although I understand the need for this, it feels odd to have an install within a test vs a build requirement.  (That said, I use the output from this test while working with composer... this is useful and convenient.)

Reply:
In order to get bf you already need to have npm installed and this is the only way I knew to ensure you have the latest.